common/test/acceptance/pages/lms/staff_view.py

```diff
@@ -51,7 +51,7 @@ def set_staff_view_mode_specific_student(self, username_or_email):
         """
         Set the current preview mode to "Specific Student" with the given username or email
```

Comment:
Nit: can you update this doc string too (OK to leave method name the way it is)?